src/Core/src/Platform/iOS/KeyboardAutoManagerScroll.cs

```diff
@@ -364,8 +364,8 @@ internal static void AdjustPosition()
 		bool cursorTooHigh = false;
 		bool cursorTooLow = false;
 
-		// Find the next parent ScrollView that is scrollable
-		var superView = View.FindResponder<UIScrollView>();
+		// Find the nearest scrollable area
```

Comment:
The test `HideSoftInputOnTappedPageTest("Editor",True)` is failing on iOS and Catalyst:
![image](https://github.com/dotnet/maui/assets/6755973/bd8eee1b-3af3-45eb-b133-e60605c0241d)

Reply:
@jsuarezruiz this should now be fixed